.golangci.yml

```diff
@@ -214,3 +214,6 @@ issues:
     - path: (pd-analysis|pd-api-bench|pd-backup|pd-ctl|pd-heartbeat-bench|pd-recover|pd-simulator|pd-tso-bench|pd-ut|regions-dump|stores-dump)
       linters:
         - errcheck
+  include:
+    # remove the comment after the path is ready
```

Comment:
what does this `path` mean?

Reply:
After adding comments to all exported name/function, these two lines of comments will be removed.

Reply:
This PR is just a partial part.

Reply:
got it

Reply:
maybe can linked with #8440 in pr description :-)

Reply:
No problem. BTW, it has been linked to the issue